source/MRMesh/MRPointCloud.cpp

```diff
@@ -49,6 +49,21 @@ void PointCloud::addPartByMask( const PointCloud& from, const VertBitSet& fromVe
     invalidateCaches();
 }
 
+void PointCloud::addPoint(const Vector3f& point)
+{
+    VertId id(points.size());
+    points.push_back(point);
+    validPoints.autoResizeSet(id);
```

Comment:
please check here that normals is empty, if it is not add warning and add zero normal